### PR TITLE
[Mri violations]Update react table mri protocol violations - rebase

### PR DESCRIPTION
**Pull request by kongtiaowang**

### CODE REVIEW

File: modules/mri_violations/jsx/mri_protocol_violations_columnFormatter.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/* exported formatColumn */
```
Comment (alisterdev) [inline modules/mri_violations/jsx/mri_protocol_violations_columnFormatter.js]: Rename this and compiled file to `columnFormatterProtocolViolation.js` to correspond to the existing format

**Comment by alisterdev** [comment]
@MounaSafiHarab You seemed to have `Could not identify scan type` uploads. Can you give it a quick manual test?

**Comment by MounaSafiHarab** [comment]
@kongtiaowang 

I am adding the discussion required just to see if 
1) the Link under Patientname (now taking us to dicom_archive) makes sense; this feature was not in the previous release, and 
2) if it makes sense to link the column just before it to BrainBrowser like we do on the main landing page of MRI Violations
mri_violations

**Comment by MounaSafiHarab** [comment]
@kongtiaowang 

I actually found a problem with the url with the PatientName and SeriesUID being undefined, although I clicked on specific row in the mri violations landing page.
url_mriviolations

**Comment by kongtiaowang** [comment]
@MounaSafiHarab 
The PatientName and SeriesUID are undefined, because of the URL where you input = "https://wangshen-dev.loris.ca/mri_violations/?submenu=mri_protocol_violations&PatientName=undefined&SeriesUID=undefined"

**Comment by MounaSafiHarab** [comment]
@kongtiaowang 

it is all good now.
The one thing I noticed, the data (PatientName and SeriesUID)  is now properly populated but I need to hit the "Show Data" button to apply the filter. Shouldn't the filter be applied automatically?

**Pull request merged**